<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneLayerShift</class>
 <widget class="QDialog" name="SceneLayerShift">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2" colspan="3">
    <widget class="QLabel" name="entShift">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Entity shift:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="Line" name="VeryUsefulLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="shiftEntBtn">
     <property name="text">
      <string>Shift entities position</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="keepDimensionsBtn">
     <property name="text">
      <string>Keep layer dimensions</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="AcceptCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="shiftY"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="vShiftLabel">
     <property name="text">
      <string>Vertical Shift</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="layerTypeLabel">
     <property name="text">
      <string>Tile Layer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="shiftX"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="hShiftLabel">
     <property name="text">
      <string>Horizontal Shift</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QLabel" name="layerDimension">
     <property name="text">
      <string>Layer Dimensions:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="layerName">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AcceptCancel</sender>
   <signal>accepted()</signal>
   <receiver>SceneLayerShift</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AcceptCancel</sender>
   <signal>rejected()</signal>
   <receiver>SceneLayerShift</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
